### PR TITLE
added segmentations for witvliet2020 and empty templates

### PR DESCRIPTION
**Pull request by sandyhider**
Added the new segmentations to public and the empty templates as well.   Also added the empty template for microns pinky100

### CODE REVIEW

File: django/bosscore/views/views_resource.py [hunk 1]
```diff
@@ -656,7 +656,8 @@ def get(self, request, collection, experiment, channel):
                                   962, 963, 964, 965, 966, 967, 975, 976, 977, 978, 979, 980, 981, 982, 983, 984, 985,
                                   986, 987, 988, 989, 993, 995, 997, 1004, 1033, 1034, 1035, 1037, 1039, 1040, 1041,
                                   1042, 1043, 1044, 1045, 1046, 1047, 1062, 1066, 1067, 1068, 1069, 1070, 1071, 1072,
-                                  1073, 1074, 1080, 1083] or request.user.has_perm("read", channel_obj):
+                                  1073, 1074, 1080, 1083, 1090, 1091, 1092, 1093, 1094, 1095, 1096, 1097, 1098, 1099,
```
Comment (movestill) [inline django/bosscore/views/views_resource.py:659]: Now that this is _this_ huge, we should put this in a `frozenset` for faster lookup and define the `frozenset` in a single file so we don't have to worry about mismatches between these two files.

https://docs.python.org/3.8/library/stdtypes.html#set-types-set-frozenset

**Review by movestill — CHANGES_REQUESTED** [review]

File: django/bosscore/views/views_resource.py (hunk 1 above)
Comment (sandyhider) [reply django/bosscore/views/views_resource.py]: Updated to use a frozenset.

File: django/bosscore/views/views_resource.py [hunk 2]
```diff
@@ -28,13 +28,16 @@
...
 from bosscore.privileges import check_role
+from bosscore.public_channels import PUBLIC_CHANNELS
 
 from bosscore.serializers import CollectionSerializer, ExperimentSerializer, ChannelSerializer, \
     CoordinateFrameSerializer, CoordinateFrameUpdateSerializer, ExperimentReadSerializer, ChannelReadSerializer, \
     ExperimentUpdateSerializer, ChannelUpdateSerializer, CoordinateFrameDeleteSerializer
 
 from bosscore.models import Collection, Experiment, Channel, CoordinateFrame, Source
 
+public_channel_frozen_set
```
Comment (movestill) [inline django/bosscore/views/views_resource.py:39]: Looking at this on my phone but this looks like a mistake?

Comment (sandyhider) [reply django/bosscore/views/views_resource.py]: Nice catch, which was my first idea and I decided to go with a separate file. I removed it.

File: django/bosscore/public_channels.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+os.chdir(os.path.abspath(os.path.dirname(__file__)))
```
Comment (movestill) [inline django/bosscore/public_channels.py:17]: I'm kind of worried about changing the working directory. Instead can you just provide the absolute path to `open()`?

```
channels_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'public_channels.txt')
```

**Review by movestill — APPROVED** [review]

File: django/bosscore/public_channels.py (hunk 3 above)
Comment (sandyhider) [reply django/bosscore/public_channels.py]: I like it!

**Pull request merged**